### DISCUSSION TITLE
PowerPoint table of contents

### DISCUSSION THREAD

**Discussion by nc011** [davidgohel/officer#578] [Q&A]
Hi,

Is there an automated way to produce a table of contents in a PowerPoint file in officer? Obviously this could be done manually, but the slide numbers may change as the presentation is updated.

Is there a way to assign a unique identifier to each slide and to find its slide number? 

Thanks!

**Comment by markheckmann** [comment]
Creating a TOC is no standard feature in PPT. What would it look like? How would the names and levels of the TOC be determined?
